.github/workflows/release.yaml

```diff
@@ -0,0 +1,33 @@
+name: Upload Release .deb
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  eb:
+    name: Upload Release .deb
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
This is a fairly alarming permission since it does give the workflow permission to make commits in to the repository. But, to upload a release asset we need this.